### DISCUSSION TITLE
issuing office for alert

### DISCUSSION THREAD

**Discussion by JJKraw** [weather-gov/api#713] [Q&A] [answered]
I'd like to be able to access the office information for any alert that I'm processing. Currently, the alerts contain "senderName" but that's a string and doesn't give me the three letter code I need to do an API request to get what I want.

It looks like it could be scraped from a couple parameters but I don't know if this is consistent across all alerts.

```
        "parameters": {
            "AWIPSidentifier": [
                "SVRTSA"
            ],
            "WMOidentifier": [
                "WUUS54 KTSA 012049"
            ],
```

In this case the ID is "TSA" (Tulsa, OK).

Is it possible to use the code as pulled from one of these parameters? Is it somewhere else that I'm just missing? If the answer to both is no, can I throw an enhancement request onto the pile to add the issuing office code or URL to the JSON returned for an alert?

**Comment by sullynole** [comment, marked as answer]
Reference the CAP documentation here for more details on the fields available: https://vlab.noaa.gov/web/nws-common-alerting-protocol/cap-documentation

**Reply by JJKraw** [reply]
Excellent. What I needed:

> Format
><parameter>
><valueName>AWIPSidentifier</valueName>
><value>NNNXXX</value>
></parameter>
>
>Where:
>
>NNN is the AWIPS product category and XXX is the AWIPS office identifier.

Thanks!!
